src/_cffi_src/openssl/bio.py

```diff
@@ -130,7 +132,14 @@
 long BIO_set_buffer_size(BIO *, long);
 long BIO_set_buffer_read_data(BIO *, void *, long);
 long BIO_set_nbio(BIO *, long);
+void BIO_set_retry_read(BIO *);
+void BIO_clear_retry_flags(BIO *);
+
+#define CRYPTO_LOCK_BIO ...
 """
 
 CUSTOMIZATIONS = """
+int * Cryptography_bio_references(const BIO * b) {
```

Comment:
BIOs are opaque in 1.1.0 so this likely needs to be obtained in a different way. I see current CPython master doesn't consider them opaque though so maybe I'm wrong? cc @tiran